content/get_start/UseCase_Workflows/revit.md

```diff
@@ -0,0 +1,108 @@
+Revit is a building information modeling software used for planning, designing, constructing and managing buildings and infrastructure. The following workflow is based on the capabilities present in Revit 2024 and is likely applicable to other versions as well. 
+
+The steps to apply the PRM measure to a BIM model created in Revit are as follows:
+
+1. [**Create geometry**](#1-create-geometry)
+2. [**Define assemblies, internal loads, HVAC**](#2-define-assemblies-internal-loads-hvac)
+3. [**Export model: Different routes**](#3-export-model-different-routes)
+4. [**Apply 90.1 PRM method**](#4-apply-901-prm-method)
+
+{{< line_break >}}
+
+##### **1. Create a geometry**
+ - Create a layout of the building using the Revit elements like walls, roofs, floors, and windows based on the design stage of the model. 
+        
+    - For a Schematic Design (SD) model, a simple shoe-box model could be used to represent the model, envelope elements can be generated using the Revit conceptual massing feature and the windows could be modeled as a percentage of exterior wall area. 
+        
+    - For a Design Development (DD) model, consider separating some spaces based on their function. Model walls, roofs, floors an windows using actual Revit elements. 
+        
+    - For a Construction Document (CD) model, more detailed architectural, structural and lighting layouts should be created along with the actual building elements.
+    
+|Plan (DD stage)| 3D Model (DD stage) |
+|:-:|:-:|
+|![userdata](/BEM-for-PRM/get_start/UseCase_Workflows/images/revitplan_DDstage.png?width=500px&align=right&classes=border,alignCenter)|![userdata](/BEM-for-PRM/get_start/UseCase_Workflows/images/revit3D_DDstage.png?width=700px&align=right&classes=border,alignCenter)|
+
+
+ ##### **2. Define assemblies, internal loads, HVAC**
+- Envelope Construction: Assign envelope assemblies using either: 
+    - Generic layer-by-layer assemblies from Revit built-in library (typical for SD phase). 
+    - Schematic construction types present in the Revit build-in library (typical for DD phase).
+    - Actual construction defined in the Revit model (typical for CD phase).
+
+- Internal Loads: Assign internal loads based on the default loads present in Revit. The defaults could be based on the building typology (preferred for SD phase) or on space by space method (preferred for DD phase) according to various standards and databases (e.g., ASHRAE 90.1, ASHRAE 62.1 and CBECS data). For a CD phase, additionally, the lighting definitions are based on the actual modeled luminaires. 
+
+- Thermal Zones: Assign the thermal zones. It could be single thermal zone for each floor of the building for an SD stage model or based on the core/perimeter layout for DD and CD stage models. A single thermal zone may encompass several spaces. 
+
+- HVAC: Assign an HVAC system. 
+    - For an SD model, HVAC system could be skipped since an ideal load air system system automatically generated in the export process. 
```

Comment:
I think we should include a note clarifying that for the PRM measure to work, the model must contain an actual HVAC system. The PRM measure will not work if the model is set up with an ideal air load system.

Reply:
We do have a note in the section "Apply 90.1 PRM measure" that says: 

"HVAC System Integration

One of the key data elements often missing in the initial stage of design is related to the HVAC systems. For example, ideal load air system commonly defined for SD phase since a complete HVAC system is not typically necessary. Additionally, various export methodologies might not be able to translate the HVAC system design into energy models."

However, if we remove the SD phase completely, then we might not to add anything. I am still not sure if we should be doing so.

Reply:
Design phases have been removed due to the restructuring of the narrative, so this should be resolved.